### ISSUE TITLE
riot-desktop hangs on initial loading screen

### ISSUE THREAD

**Issue by lsaether** [NixOS/nixpkgs#76976]
On nearly a fresh install of 19.09, the `riot-desktop` package installs and hangs at the loading screen. Even after waiting for 20 minutes the app never loads. There doesn't seem to be any errors being thrown in the development console (See screenshot below).

![image](https://user-images.githubusercontent.com/25539605/71778860-7d88f300-2fb3-11ea-8d77-bb258ada4d62.png)

**Comment by worldofpeace** [comment]
I'm considering we need to backport updates to riot-desktop and riot-web to stable.

**Comment by worldofpeace** [comment]
@pacien

**Comment by pstn** [comment]
Yes, please backport. I know at least one person being blocked by this.

**Comment by gcoakes** [comment]
I'm having what looks like a similar issue. What commit fixes this? I'm running the unstable branch and just updated but am still affected.

**Comment by pacien** [comment]
This upstream issue seems relevant: https://github.com/vector-im/riot-web/issues/7730

**Comment by worldofpeace** [comment]
I change was backported to 19.09 https://github.com/NixOS/nixpkgs/pull/80393, that hopefully fixed this without having to do a bunch of invasive updates. One can test with
```
$ nix run -f https://github.com/NixOS/nixpkgs/archive/483f61d1506fdb744f98acc402b94e1eb33421b8.tar.gz riot-desktop

$ riot-desktop
```

**Comment by lsaether** [comment]
No longer relevant to me. Thanks for the help!
